go.mod

```diff
@@ -252,7 +262,6 @@ replace (
 	go.opentelemetry.io/otel/sdk/metric => go.opentelemetry.io/otel/sdk/metric v0.20.0
 	go.opentelemetry.io/otel/trace => go.opentelemetry.io/otel/trace v0.20.0
 	go.opentelemetry.io/proto/otlp => go.opentelemetry.io/proto/otlp v0.7.0
-	golang.org/x/net => golang.org/x/net v0.0.0-20210224082022-3d97a244fca7
```

Comment:
Whilst it is unlikely to cause an issue, we've not had a release go out to the world yet that actually includes the upgrade to this package right? I'm a _little_ nervous we don't have any feedback that it doesn't cause problems (that could be picked up in an alpha release for 1.10 for example) and we're backporting this into 1.9 'blind'.

I don't think it's a _massive_ deal but given we don't think we're actually vulnerable to exploit through these older dependencies, it seems like additional risk with little to no gain from an end user perspective.

I'm pretty indifferent really, but thought I'd call it out :) (same goes for go-restful.. whilst we may not be vulnerable to exploits, the dependency upgrade _could_ cause a material difference to behaviour still)